LATEST

```diff
@@ -1 +1 @@
-0.0.1-141-gd8ef7c3
+0.0.1-46-g368774f
```

Comment:
This is a regression that would undo a041950 and potentially undo #283 if that gets merged first.